frontend/src/components/LandingBody.tsx

```diff
@@ -0,0 +1,66 @@
+import { Container, Button } from 'react-bootstrap';
+import { useState, ReactNode } from 'react';
+import { LinkButton } from './LinkButton';
+import { Badges } from './Badges';
+import './LandingBody.css';
+interface ILandingBodyProps {
+
```

Comment:
Right now the landing page looks really dense of text.

I think it would be cooler to have the entire frontend with tighter lines (Ex. the same text gets spread in many tighter lines). 

In this way the landing page would seem leaner I guess